test/helpers/amman.ts

```diff
@@ -1,6 +1,55 @@
 import { Amman } from '@metaplex-foundation/amman';
-import { PROGRAM_ADDRESS as TOKEN_METADATA_ADDRESS } from '@metaplex-foundation/mpl-token-metadata';
+import {
+  PROGRAM_ADDRESS as TOKEN_METADATA_ADDRESS,
+  cusper as cusperTokenMetadata,
+} from '@metaplex-foundation/mpl-token-metadata';
+import { TransactionError } from '@solana/web3.js';
 
 export const amman = Amman.instance({
   knownLabels: { [TOKEN_METADATA_ADDRESS]: 'Token Metadata' },
 });
+
+// -----------------
+// Amman Utils
+// -----------------
+
+// Most of the below should get added to amman or cusper proper
```

Comment:
I want the SDK to support these error resolutions out of the box so we shouldn't even need these helpers in the future. But glad to see them here for now! :)